### PR TITLE
Bump up to v0.3.3

### PR DESCRIPTION
**Pull request by kampersanda**

### CODE REVIEW

**Comment by vbkaisetsu** [comment]
I'll approve this after #105 is merged

**Comment by kampersanda** [comment]
@vbkaisetsu Ah, sorry. I was mistaken about https://github.com/daac-tools/vibrato/pull/105. I see.

**Review by vbkaisetsu — APPROVED** [review]

**Pull request merged**
